scripts/nightly.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
I can run or whenever @hongchaodeng needs a new image.

Quay has github trigger, if we need more automation.